docuilib/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "uilib-docs",
-  "version": "3.7.0",
+  "version": "3.8.0",
```

I just recently bump myself to 3.8.0 in another PR
```suggestion
  "version": "3.9.0",
```